pylint/checkers/variables.py

```diff
@@ -2310,9 +2310,7 @@ def _check_is_unused(self, name, node, stmt, global_names, nonlocal_names):
             if global_names and _import_name_is_global(stmt, global_names):
                 return
 
-        argnames = list(
-            itertools.chain(node.argnames(), [arg.name for arg in node.args.kwonlyargs])
-        )
+        argnames = node.argnames()
```

Comment:
We discussed leveraging the changes in astroid to simplify this here: https://github.com/PyCQA/pylint/issues/5771#issuecomment-1030703199

Reply:
Pretty nice simplification !